README.md

```diff
@@ -66,42 +83,23 @@ To run the node you'll need to provide a configuration file. We have an example
 miden-node start --config <path-to-config-file>
 ```
 
-Or, if your config file is in named `miden.toml` and is in the current directory, you can simply run:
+Or, if your config file is named `miden.toml` and is in the current directory, you can simply run:
```

Comment:
change `current` to `root`

Reply:
I've just tested and it searches for the file `miden.toml` in the current directory, not in the root.